### DISCUSSION TITLE
Hidden Web pages on ODI DFP-34X-2C2

### DISCUSSION THREAD

**Discussion by aoeII** [Anime4000/RTL960x#38] [General]
Hi all,  by accident I got a list of most of the asp pages and others scripts files that might work on ODI DFP-34X-2C2 :

acl.asp
addr_mapping.asp
admin
adslpsd.asp
adsltone.asp
algonof.asp
app_iptv.asp
app_mld_snooping.asp
app_mldProxy.asp
arptable.asp
base.css
base64_code.js
bridge_grouping.asp
bridging.asp
captcha.gif
captcha.html
code.asp
common.js
config_802_1x.asp
connlimit.asp
date.asp
ddns.asp
devicetbl.asp
dhcpd.asp
dhcpd_sc.asp
dhcpdv6.asp
dhcpmode.asp
dhcptbl.asp
dhcptblv6.asp
diag-test.asp
differv.asp
dms.asp
dns.asp
domainblk.asp
dos.asp
dot1ag_action.asp
dot1ag_conf.asp
dot1ag_status.asp
EndScriptContent.asp
epon.asp
eth2pvc.asp
eth2pvc_vlan.asp
ethoam.asp
favicon.ico
fdbtbl.asp
fnishmaintenance.asp
fw-dmz.asp
fw-ipfw.asp
fw-ipportfilter.asp
fw-ipportfilter_rg.asp
fw-ipportfilter-v6.asp
fw-ipportfilter-v6_lfld.asp
fw-ipportfilter-v6_lfld_rg.asp
fw-ipportfilter-v6_rg.asp
fw-macfilter.asp
fw-macflter_rg.asp
fw-porfw.asp
gaming.asp
gpon.asp
osgi_bndins.asp
osgi_bndmgt.asp
osgi_fwork.asp
others.asp
parental-ctrl.asp
password.asp
pdump.asp
ping.asp
portBaseFilterDhcp.asp
portw-advance.asp
pptp.asp
pptpd.asp
pptpv6.asp
printServer.asp
qos_pred.asp
radvdconf.asp
reboot.asp
rebootTime.asp
reset.css
rip.asp
rmtacc.asp
routetbl.asp
routetbl_ipv6.asp
routing.asp
routing_ipv6.asp
samba.asp
saveconf.asp
share.js
snmp.asp
snmpv3.asp
StartScriptContent.asp
stats.asp
status.asp
status_ipv6.asp
status_pon.asp
style
style.css
syslog.asp
syslog_server.asp
tcpiplan.asp
topbar.gif
toupgrade.asp
tr069config.asp
tz.asp
upgrade.asp
upladlogo.asp
upnp.asp
url_blocking.asp
userconfig.asp
~~vermod.asp~~
vlan.asp
vlan_on_lan.asp
wan3gconf.asp
wanadsl.asp
wanatm.asp
waneth.asp
wanmode.asp
wanwl.asp
web_hijackwizard_nolink.asp
web_hijackwizard_wait1.asp
web_hijackwizard_wait2.asp

I haven't enough time to test them all yet. 

~~Enjoy it.~~

update : I am very **sorry** for that the OMCI Export function in vermod.asp would make the stick brick. So forget it please.

**Comment by stich86** [comment]
for which firmware?
looks like that "vermod.asp" is not present on latest firmware 2205XX :(

**Reply by aoeII** [reply]
> for which firmware? looks like that "vermod.asp" is not present on latest firmware 2205XX :(

Hi stich, the link for `vermod.asp` is valid on 220414 firmware. I have not flashed 2205XX firmware to my stick as Anime said it is not a bridge sfu firmware.

**Reply by Anime4000** [reply]
`vermod.asp` doesn't do anything beside export OMCI that cause my ODI stick brick!

**Reply by vuducdong** [reply]
@Anime4000 `vermod.asp` I think it's just in test build and now only effective in quick change for GPON SN and ELAN MAC Address. 

if you try to use OMCI export it will crash sfp, then you need to restart sfp to use it normally.

**Reply by aoeII** [reply]
> `vermod.asp` doesn't do anything beside export OMCI that cause my ODI stick brick!

Hi Anime, I am sorry for that. Has it recovered to nomal status after pulled out and plugged in again?

**Reply by Anime4000** [reply]
nope, it's broken 😢 
currently I try restore bin back to SPI Flash

**Reply by aoeII** [reply]
> nope, it's broken 😢 currently I try restore bin back to SPI Flash

I am very **sorry** for that the OMCI Export function in vermod.asp would make the stick brick. 
Good luck !

**Reply by Anime4000** [reply]
yea, hahaha, UART Terminal didn't show anything, SoC I have is officially dead, overheat to 110°C in status page

well this not first time I destroy ONU Stick 🤣
my V2801F just had blown laser modue 🤣🤣🤣

**Comment by stich86** [comment]
here is the list of 2205XX asp page, funny thing: there is a page (`uploadlogo.asp`) to upload custom logo  :)

```
# ls *.asp
acl.asp                         password.asp
app_iptv.asp                    password_first.asp
arptable.asp                    ping.asp
bridging.asp                    ponmodecfg.asp
captcha.asp                     portBaseFilterDhcp.asp
code.asp                        radvdconf.asp
ddns.asp                        reboot.asp
dhcpd.asp                       rebootTime.asp
dhcpdv6.asp                     rip.asp
dhcptbl.asp                     routetbl.asp
dhcptblv6.asp                   routetbl_ipv6.asp
diag_ping_admin_result.asp      routing.asp
dos.asp                         routing_ipv6.asp
epon.asp                        saveconf.asp
fdbtbl.asp                      stats.asp
fw-dmz.asp                      status.asp
fw-ipportfilter-v6_IfId_rg.asp  status_ipv6.asp
fw-ipportfilter_rg.asp          status_pon.asp
fw-macfilter_rg.asp             tcpiplan.asp
fw-portfw.asp                   top.asp
gpon.asp                        tr069config.asp
ipv6_enabledisable.asp          tz.asp
lbd.asp                         upgrade.asp
macIptbl.asp                    uploadlogo.asp
multi_lang.asp                  upnp.asp
multi_wan_generic.asp           vlan.asp
omci_info.asp
```
